pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTest.java

```diff
@@ -364,18 +367,32 @@ protected long getCurrentCountStarResult()
    * @param timeoutMs Timeout in milliseconds
    * @throws Exception
    */
-  protected void waitForAllDocsLoaded(long timeoutMs)
-      throws Exception {
+  protected void waitForAllDocsLoaded(long timeoutMs) throws Exception {
     final long countStarResult = getCountStarResult();
     TestUtils.waitForCondition(new Function<Void, Boolean>() {
       @Nullable
       @Override
       public Boolean apply(@Nullable Void aVoid) {
+        long currentResult;
         try {
-          return getCurrentCountStarResult() == countStarResult;
+          currentResult = getCurrentCountStarResult();
         } catch (Exception e) {
           return null;
         }
+
+        // Add more information to logs
+        if (currentResult == countStarResult) {
+          return true;
+        } else if (currentResult < countStarResult) {
+          LOGGER.info("Docs are not fully loaded yet (current: {}, expected: {})", currentResult, countStarResult);
```

Comment:
We disable info-level logging in our Travis CI setting. So in Travis it should be fine. The only concern is the place where info level logging is enabled.

Reply:
In that case, let us change this particular waitForCondition call to be every 10 seconds.